conf/modules/nav_skid_landing.xml

```diff
@@ -19,16 +19,54 @@ Block example:
        <define name="FINAL_HEIGHT" value="50" unit="m"/>
        <define name="FINAL_STAGE_TIME" value="10" unit="s"/>
      </section>
-</block>
 @endverbatim
 -->
+
+Also define 
+<!--
```

Comment:
if you comment these things out, they won't appear in the generated docs: http://docs.paparazziuav.org/latest/module__nav_skid_landing.html

You can instead make the complete `<section>` part of the `<doc>` node...